st2common/st2common/jinja/filters/time.py

```diff
@@ -56,7 +57,7 @@ def _get_human_time(seconds):
         return '%s\u03BCs' % seconds  # Microseconds
 
     if isinstance(seconds, float):
-        seconds = long(round(seconds))  # Let's lose microseconds.
+        seconds = int(round(seconds))  # Let's lose microseconds.
```

Comment:
Is st2 going py3 only? Otherwise, should check for which py version and keep the long for py2.x.

Reply:
Fixed in a3f1c0908